### DISCUSSION TITLE
Realtime data

### DISCUSSION THREAD

**Discussion by paradis-A** [directus/directus#10608] [Archive]
I saw the realtime data in the roadmap. It would be great if directus have that functionality. Anyway is there a branch where we can test it?

**Comment by benhaynes** [comment]
No branch yet, still in r&d.

**Comment by rijkvanzanten** [comment]
If you wanna experiment with how it could work, you should give @br41nslug's extension a try! https://github.com/br41nslug/directus-websocket-subscribe

**Comment by 3zzy** [comment]
Is it available yet? New to Directus, so just wondering if there's way of knowing what things from the roadmap have been released or their status? Thanks!

**Reply by benhaynes** [reply]
Not yet, but in the works. We will actually be updating the roadmap to not give any timeline. The only accurate way to get a release time is to sponsor a feature/ fix.
